### PR TITLE
Bulk load igv

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: ui/pages/DataManagement/reducers.js [hunk 1]
```diff
@@ -53,29 +54,49 @@ export const uploadQcPipelineOutput = values => submitRequest(
 
 export const deleteEsIndex = index => submitRequest('delete_index', RECEIVE_ELASTICSEARCH_STATUS, { index })
 
-export const uploadRnaSeq = values => (dispatch) => {
+const loadMultipleData = (path, getUpdateData, dispatchType, formatSuccessMessage) => values => (dispatch) => {
   let successResponseJson = null
   return new HttpRequestHelper(
-    '/api/data_management/update_rna_seq',
+    `/api/data_management/${path}`,
```
Comment (ShifaSZ) [inline ui/pages/DataManagement/reducers.js:60]: Moving the full URL into the path parameter can make the function easier to be used for more cases.

File: seqr/views/apis/igv_api.py [hunk 2]
```diff
@@ -75,11 +73,58 @@ def _process_alignment_records(rows, **kwargs):
...
+        return uploaded_file_id, None, records
+
+    def _get_valid_matched_individuals(individual_dataset_mapping):
+        individuals = Individual.objects.filter(
+            family__project__guid__in=get_project_guids_user_can_view(request.user, limit_data_manager=False),
+            family__project__name__in={k[0] for k in individual_dataset_mapping.keys()},
+            individual_id__in={k[1] for k in individual_dataset_mapping.keys()},
+        ).select_related('family__project')
+        individuals_by_project_id = {(i.family.project.name, i.individual_id): i for i in individuals}
+        unmatched = set(individual_dataset_mapping.keys()) - set(individuals_by_project_id.keys())
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/igv_api.py:118]: In the scenario where IGV is being updated within a single project, there is no advantage to the tuple as, by definition, the project is always the same. As such, it is simpler to maintain the previous behavior in that case, and only add the project-individual tuple in cases where there are actually multiple projects, which is what this PR does

File: ui/pages/DataManagement/components/AddIGV.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <br />
+          Columns are as follows:
+          <br />
+          <List ordered>
+            <List.Item>Project</List.Item>
+            <List.Item>Individual ID</List.Item>
+            <List.Item>IGV Track File Path</List.Item>
+            <List.Item>
+              Optional: Sample ID if different from Individual ID.
+              Used primarily for gCNV files to identify the sample in the batch path
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/pages/DataManagement/components/AddIGV.jsx:30]: This look good the way it is

**Review by ShifaSZ — APPROVED** [review]
LGTM

**Pull request merged**